console/algorithms/src/bhp/hasher/mod.rs

```diff
@@ -81,7 +81,8 @@ impl<E: Environment, const NUM_WINDOWS: u8, const WINDOW_SIZE: u8> BHPHasher<E,
         }
 
         // Compute the bases lookup.
-        let bases_lookup = cfg_iter!(bases)
+        let bases_lookup = bases
```

Comment:
This was a subtle issue. I'm not sure if this can be run in parallel, but due to the previous mix of default & parallel features, this was previously always being run serially so I thought to make it explicit.

Reply:
I'd actually suggest just leaving it alone. I understand you're going for code hygiene and this is good, but unless we're certain it can't be run in `parallel` it should be left as-is to minimize side effects.

Reply:
We should preserve cfg_iter! on this.

Reply:
Hm I would benchmark and see whether parallelization is beneficial here; it might be the case that parallelization introduces too much overhead.